<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchPannel</class>
 <widget class="QDialog" name="SearchPannel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>搜索面板</string>
  </property>
  <widget class="QLabel" name="timeLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;时间:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>194</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="locationLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>地点:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="provinceLineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="provinceLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>省</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cityLineEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="cityLabel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>100</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>市</string>
   </property>
  </widget>
  <widget class="QLabel" name="blockLabel">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>100</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>区/县</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="blockLineEdit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>100</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="detailLocLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>详细地址：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="detailLocLineEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="facesLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>人脸(以“/”分隔）</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="facesLineEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>210</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="labelLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>标签(以“/”分隔）</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lableLineEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>260</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>330</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>330</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
